com.civicrm.hrjobroles/CRM/Hrjobroles/Import/Form/DataSource.php

```diff
@@ -60,8 +58,12 @@ public function preProcess() {
   public function buildQuickForm() {
     //Setting Upload File Size
     $config = CRM_Core_Config::singleton();
-
-    $uploadFileSize = CRM_Core_Config_Defaults::formatUnitSize($config->maxFileSize.'m');
+    if ($config->maxImportFileSize >= 8388608) {
+      $uploadFileSize = 8388608;
+    }
+    else {
+      $uploadFileSize = $config->maxImportFileSize;
+    }
```

Comment:
I did this change due to your comment on this PR :

>I tried to run this in 4.7.7 and got an error, because CRM_Hrjobroles_Import_Form_DataSource is using CRM_Core_Config_Defaults which was removed in 4.7 (civicrm/civicrm-core@d7c217a)

so this remove the dependency on CRM_Core_Config_Defaults  which is part of you suggestions . Also I am going to squash  ( "PR Review Suggestions") commit before merging.

for 8388608 to be honest  I saw it implemented in https://github.com/civicrm/civihr/blob/07576b915e22cc64a1b5211cc7916bef36f01667/hrjob/CRM/HRJob/Import/Form/DataSourceBaseClass.php and https://github.com/civicrm/civihr/blob/07576b915e22cc64a1b5211cc7916bef36f01667/uk.co.compucorp.civicrm.appraisals/CRM/Appraisals/Import/Form/DataSourceBaseClass.php 

so I copied it to insure consistency among importers code , personally I don't see a big value for it rather than limiting the size of imported file to a certain value regarding the configs which maybe be part of some old requirements . do you think I should keep or remove it ?